src/rdkafka_conf.c

```diff
@@ -1890,6 +1899,32 @@ void rd_kafka_conf_set_open_cb (rd_kafka_conf_t *conf,
 }
 #endif
 
+rd_kafka_conf_res_t
+rd_kafka_conf_set_ssl_cert_verify_cb(rd_kafka_conf_t *conf,
+    int (*ssl_cert_verify_cb) (char *cert, size_t len, void *opaque)) {
+#ifdef __mips__
```

Comment:
I think this should read: `if defined(__mips__) || !WITH_SSL`, we want to return UNKNOWN in both those cases